index.html

```diff
@@ -20,377 +20,273 @@
-         <div class="second-tag3">
-          <span>Ruby</span>
-         </div>
-        </div>
-
-        <div class="action">
-          <h3>See Project</h3>
-        </div>
-
-
-
 
-        
-
-      </section>
-      <section class="projects">
-
-        
-        <h1>Profesional Art Printing Data</h1>
-        <small>A daily selection of privately personalized reads; 
-          no accounts or sign-ups required. 
-          has been the industry's standard
-        </small>
-        <div class="third-tags">
-          <div class="third-tag">
-           <span>Html</span>
-          </div>
-          <div class="third-tag2">
-           <span>bootstrap</span>
-          </div>
-          <div class="third-tag3">
-           <span>Ruby</span>
-          </div>
-         </div>
-
-        <div class="action">
-          <h3>See Project</h3>
-        </div>
-
-
-
-
-        
-
-      </section>
-      <section class="projects">
-
-        
-        <h1>Profesional Art Printing Data</h1>
-        <small>A daily selection of privately personalized reads; 
-          no accounts or sign-ups required. 
-          has been the industry's standard
-        </small>
-        <div class="fourth-tags">
-          <div class="fourth-tag">
-           <span>Html</span>
-          </div>
-          <div class="fourth-tag2">
-           <span>bootstrap</span>
+      <div class="card-works">
+        <div class="feature">
+          <div class="img-placeholder">
+            <img src="assets/img-placeholder.png" alt=""/>
           </div>
-          <div class="fourth-tag3">
-           <span>Ruby</span>
-          </div>
-         </div>
-
-        <div class="action">
-          <h3>See Project</h3>
-        </div>
-
-
-
-
-        
-
-      </section>
-      <section class="projects">
-
-        
-        <h1>Profesional Art Printing Data</h1>
-        <small>A daily selection of privately personalized reads; 
-          no accounts or sign-ups required. 
-          has been the industry's standard
-        </small>
-        <div class="fifth-tags">
-          <div class="fifth-tag">
-           <span>Html</span>
-          </div>
-          <div class="fifth-tag2">
-           <span>bootstrap</span>
-          </div>
-          <div class="fifth-tag3">
-           <span>Ruby</span>
+          <div class="right-block">
+            <h1>Multi-Post Stories</h1>
+            <p>A daily selection of privately personalized reads; 
+              no accounts or sign-ups required. 
+              has been the industry's standard dummy text ever since the 1500s, 
+              when an unknown printer took a standard dummy text.
+            </p>
+            <div class="tag">
+              <ul>
+                <li>Css</li>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
+              </ul>
+            </div>
+            <div class="block-btn">
+              <h4>See Project</h4>
+            </div>
```

Comment:
- [ ] This `See project` is the buttons, I would suggest you please use the button tag or `a` tag for better semantics HTML.
- [ ] As per the project requirement, please implement the `hovered`, and `Pressed functionality. See the image below for reference.
### Design
![Screenshot 2022-12-09 130627](https://user-images.githubusercontent.com/81727988/206647956-8fa0a027-1da5-42c4-ba03-8ba5a4c41fa5.png)